Dockerfile-test

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.11.2-alpine
+
+ENV CGO_ENABLED=0
+ENV GO111MODULE=on
+
+WORKDIR /go/src/github.com/AirHelp/filler
+RUN apk add --no-cache git
+
+COPY . .
```

Comment:
it downloads all the dependencies each time, you can avoid that by using 
```suggestion
# We want to populate the module cache based on the go.{mod,sum} files.
COPY go.mod .
COPY go.sum .

#That will download all the dependencies that are specified in the go.mod and go.sum file.
# Because of how the layer caching system works in Docker, the  go mod download 
# command will _ only_ be re-run when the go.mod or go.sum file change 
RUN go mod download

COPY . .
```